modules/cluster.py

```diff
@@ -115,13 +115,17 @@ def get_best_cluster(read_cl_id, compressed_seq_len, hit_clusters_ids, hit_clust
                 else:
                     total_mapped += compressed_seq_len - minimizer_hit_positions[-1]
 
-                mapped_ratio = total_mapped /float(compressed_seq_len) 
-                
-                if mapped_ratio > args.mapped_threshold:
-                    # is_covered = True
+                mapped_ratio = total_mapped /float(compressed_seq_len)
+
+                # Calculate the ratio of mapped region of the representative
+                rep_mapped_ratio = total_mapped / float(len(rep_compressed_seq))
+
+                if args.symmetric_map_align_thresholds and min(mapped_ratio, rep_mapped_ratio) > args.mapped_threshold:
+                    return cl_id, nm_hits, mapped_ratio
```

Comment:
Wouldn't you want to return `min(mapped_ratio, rep_mapped_ratio)` here instead of `mapped_ratio`?

Reply:
Thank you, good suggestion.  I will update this PR.  Note that it appears that the returned value of mapped_ratio is currently unused in the caller.

Reply:
I see, and same for `alignment_ratio` below. Nevertheless it might be more consistent like this. Thanks again. merging now.